mooncake-store/src/master_service.cpp

```diff
@@ -174,6 +182,9 @@ ErrorCode MasterService::PutStart(
         return ErrorCode::INVALID_PARAMS;
     }
 
+    LOG(INFO) << "### LRU Update in Put() ###";
```

Comment:
Would it make sense to use VLOG here instead? Otherwise it might end up printing a lot of unnecessary logs.